### PR TITLE
Update config var description docs

### PR DESCRIPTION
**Pull request by eranelbaz**
### Issue & Steps to Reproduce / Feature Request
[//]: <> (choose “resolves” “fixes” or “closes” and put link for the issue)
Update description

### CODE REVIEW

File: docs/resources/configuration_variable.md [hunk 1]
```diff
@@ -34,10 +35,11 @@ resource "env0_configuration_variable" "drop_down" {
 ### Required
 
 - **name** (String) name to give the configuration variable
-- **value** (String) value for the configuration variable
+- **value** (String, Sensitive) value for the configuration variable
 
 ### Optional
 
+- **description** (String) a description of the variables
```
Comment (omry-hay) [inline docs/resources/configuration_variable.md:42]: Please add that we limit it to 255 chars

**Comment by eranelbaz** [comment]
[Update TF description docs](https://trello.com/c/TM4NOeOE/4485-update-tf-description-docs)

**Comment by eranelbaz** [comment]
[Update TF description docs](https://trello.com/c/TM4NOeOE/4485-update-tf-description-docs)

**Review by omry-hay — COMMENTED** [review]
Small comment, also will that include the data resource as well?

File: docs/resources/configuration_variable.md [hunk 2]
```diff
@@ -34,10 +35,11 @@ resource "env0_configuration_variable" "drop_down" {
 ### Required
 
 - **name** (String) name to give the configuration variable
-- **value** (String) value for the configuration variable
+- **value** (String, Sensitive) value for the configuration variable
```
Comment (liranfarage89) [inline docs/resources/configuration_variable.md:38]: What does it mean? a boolean variable? do you think it's clear enough?

Comment (eranelbaz) [reply docs/resources/configuration_variable.md:38]: this is the value of the variable and it is sensitive
we just didn't update the docs apparently before

File: docs/resources/configuration_variable.md (hunk 1 above)
Comment (eranelbaz) [reply docs/resources/configuration_variable.md:42]: 66d474e675695744c8b07b2d6603e60d53bbae43

**Comment by eranelbaz** [comment]
@omry-hay didn't commit that file
now is
66d474e675695744c8b07b2d6603e60d53bbae43

**Review by liranfarage89 — APPROVED** [review]

**Pull request merged**